### PR TITLE
Updating welcome screen

### PR DESCRIPTION
**Pull request by mattfarina**

### CODE REVIEW

File: src/pages/General.vue [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
-      <p>
-        Welcome...
+    <div class="general">
+      <h1>
+        Welcome to Rancher Desktop
+      </h1>
+      <p>Rancher Desktop provides Kubernetes and image management through the use of a desktop application.</p>
+      <ul>
+        <li>Project Status: <i>alpha</i></li>
+        <li>Project Discussions: #rancher-desktop in <a href="#" onclick="require('electron').shell.openExternal('https://slack.rancher.io/')">Rancher Users</a> Slack</li>
```
Comment (mook-as) [inline src/pages/General.vue:13]: We should probably get a helper function for link clicking? Or even better, set up a [`will-navigate`](https://www.electronjs.org/docs/api/web-contents#event-will-navigate) event handler to open all external links externally?

The only links we'd open internally are the navigation things, which shouldn't trigger `will-navigate`, just `did-navigate-in-page`, right?

File: src/pages/General.vue [hunk 2]
```diff
@@ -50,4 +58,13 @@ export default {
 
 <!-- Add "scoped" attribute to limit CSS to this component only -->
 <style scoped>
+
+.general li {
+  margin-bottom: .5em;
+}
+
+.general p {
```
Comment (mook-as) [inline src/pages/General.vue:66]: We should get rid of the `p` styles in `_typography.css`; as far as I can tell that's not even copied from dashboard.

**Review by mook-as — CHANGES_REQUESTED** [review]
I'm okay with filing follow ups for my review items, and merging this as-is, as long as we fix the lint warnings :D

File: src/pages/General.vue (hunk 2 above)
Comment (mattfarina) [reply src/pages/General.vue:66]: brought over an update to this file that no longer has the font size change.

File: src/pages/General.vue (hunk 1 above)
Comment (mattfarina) [reply src/pages/General.vue:13]: Filed #231 to handle this afterwards

**Review by mook-as — APPROVED** [review]

**Pull request merged**